test/unit/spec/Util/BootstrapSearchSpec.js

```diff
@@ -22,6 +22,7 @@ define(['jquery', 'util/bootstrapSearch', 'base/js/namespace'], ($, BootstrapSea
         it('Should fire an input function when triggered by input', (done) => {
             const bsSearch = new BootstrapSearch($targetElem, {
                 inputFunction: () => {
+                    expect(true).toBeTruthy();
```

Comment:
add in a crappy test to ensure that code is actually running and this isn't just a false positive.

Reply:
Removed from PR